testing/test_capture.py

```diff
@@ -514,6 +514,11 @@ def test_hello(capfd):
         )
         reprec.assertoutcome(passed=1)
 
+    def test_cafd_includes_carriage_return(self, capfd):
+        print("Greetings from DOS", end="\r")
```

Comment:
When I think about it, the greeting was probably misattributed by me because DOS uses `\r\n`. Which BTW might be a good idea to test as well.